### ISSUE TITLE
Notifications not being read out by VoiceOver

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#72077]
From @PeterSaathoff-Harshfield

There are alerts that appear at the bottom right corner of the window. When VoiceOver focuses on them, they are reported as "notification toast." VoiceOver doesn't read anything more than "notification toast" for each one. They can't be interacted with, and therefore can't be read or dismissed with VoiceOver. Also, "toast" is maybe not a meaningful word for many people in this context even though programmers may know what a toast is.

Isidor: I could not really reproduce this, as when I focus the notification list I get the content nicely read out.
However I do see in code the `Notification Toast` [here](https://github.com/Microsoft/vscode/blob/feacb6dae91e49591777a40ba5a2bd5a7bfd49bc/src/vs/workbench/browser/parts/notifications/notificationsToasts.ts#L159)

@PeterSaathoff-Harshfield how do you get into a state that only "notificatino toast" is being read out?

**Comment by PeterSaathoff-Harshfield** [comment]
I would like to reproduce this and record a short video, but I don't know a way of creating notifications.

**Comment by isidorn** [comment]
@PeterSaathoff-Harshfield you can have a `.extensions.json` inside the `.vscode` folder with this content
That way you will always get a notification when you open that workspace inv vscode.

```json
{
	// See http://go.microsoft.com/fwlink/?LinkId=827846
	// for the documentation about the extensions.json format
	"recommendations": [
		"eg2.tslint"
	]
}
```
